cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/leveldb/LevelDBStreamAdmin.java

```diff
@@ -46,9 +46,8 @@ public LevelDBStreamAdmin(CConfiguration conf, LevelDBOrderedTableService servic
   @Override
   public String getActualTableName(QueueName queueName) {
     if (queueName.isStream()) {
-      //TODO: namespace the tableName (have to modify tableNamePrefix)
-      // <cdap namespace>.system.stream.<stream name>
-      return getTableNamePrefix() + "." + queueName.getSecondComponent();
+      // <cdap namespace>.system.stream.<namespace>.<stream name>
+      return getTableNamePrefix() + "." + queueName.getFirstComponent() + "." + queueName.getSecondComponent();
```

Comment:
added a jira for the code cleanup - since its not related to namespaced streams. will do in a separate PR. (otherwise this PR will delay to get merged).